src/main/resources/db/migration/V1_9__create_hearing_media_ae_table.sql

```diff
@@ -0,0 +1,7 @@
+CREATE TABLE IF NOT EXISTS moj_hearing_media_ae (
+  moj_hma_id                integer                     NOT NULL
+, moj_hea_id                integer                     NOT NULL DEFAULT 0
```

Comment:
Do we need the default here?

Reply:
:) Think I can guess why you have it now - rather than removing the NOT NULL constraint :)